test/runtests.jl

```diff
@@ -50,7 +50,9 @@ if anyerrors
     throw("Tests failed")
 end
 
-stdin = joinpath(dirname(@__FILE__), "stdin.sh")
-ENV2 = copy(ENV)
-ENV2["JULIA_HOME"] = JULIA_HOME
-run(setenv(`bash $stdin`, ENV2))
+if !is_windows()
```

Comment:
Yet we could have an abstract interface for `iscategorical`, `isnull` and `dropnull`. We've discussed this in another issue recently. I guess it could go into AbstractTables.